### DISCUSSION TITLE
not all attributes from parents elements are inherited to children elements

### DISCUSSION THREAD

**Discussion by ChristinaFaur** [vvvv/SVG#853] [Q&A]
in my example I want to inherit some attributes from parents to children if those are not explicitly set for the children.

This doesn't work by default. Is it possible to achieve this?

![image](https://user-images.githubusercontent.com/63300027/79143215-1ef6c080-7dbd-11ea-87be-a1b2ccf15962.png)

**This is what I get when I create a new SvgDocument and I insert the SvgRectangles that I have read from the svg-File.**

![image](https://user-images.githubusercontent.com/63300027/79143338-5c5b4e00-7dbd-11ea-855a-70b740da718c.png)

[V20000000 (2).txt](https://github.com/vvvv/SVG/files/4470807/V20000000.2.txt)

**Comment by H1Gdev** [comment]
I think that `Inherited` operates according to SVG specification.

I think in effect is not visible because `stroke` is not specified ?
